src/main/java/Bundle.properties

```diff
@@ -215,7 +215,7 @@ notification.email.assign.role.subject=Dataverse: You have been assigned a role
 notification.email.revoke.role.subject=Dataverse: Your role has been revoked
 
 notification.email.greeting=Hello, \n
-notification.email.welcome=Welcome to Dataverse 4.0! Please take a look around, try everything out, and contact support@dataverse.org if you have questions or feedback.
+notification.email.welcome=Welcome to Dataverse! Get started by adding or finding data. Have questions? Check out the User Guide at {0}/{1}/user/ or contact Dataverse Support for assistance.
```

Comment:
This has already been merged but rather than hard-coding "Dataverse Support" here, we **could** get it dynamically from some of the work done in #2088. This would require some backend work, of course.